openllm-python/src/openllm/conversation.py

```diff
@@ -0,0 +1,314 @@
+'''
```

Comment:
can we make this one `_conversation.py` instead? Let's make it internal for now.

Reply:
Done. For the llama testing I seem to encounter an error for the model-id checking. Can you check Slack?